backend/src/LibExecution/RuntimeTypes.fs

```diff
@@ -301,7 +306,7 @@ and DType =
   | TPassword
   | TUuid
   | TOption of DType
-  | TUserType of string * int
+  | TUserType of UserTypeName // this should probably be split into TUserEnum and TUserRecord
```

Comment:
I think it's fine to just use the name and find whether it's an enum or a record from the name.